diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -1187,29 +1190,31 @@ fun doesLambdaContainIt(lambdaNode: ASTNode): Boolean {
  */
 @Suppress("CyclomaticComplexMethod")
 internal fun isPairPropertyBackingField(propertyNode: ASTNode?, backingFieldNode: ASTNode?): Boolean {
```

Comment:
as I guess, it's expected that inputs are PSI instead of AstNode.

Reply:
I think its not pretty, to case nodes outside of function
btw, we can support 2 overloads

```
fun isPairPropertyBackingField(PsiNode, PsiNode)

fun isPairPropertyBackingField(ASTNode, ASTNode) = isPairPropertyBackingField(ASTNode.toPsi(), ASTNode.toPsi())

```